docs/maya/glue.md

```diff
@@ -0,0 +1,145 @@
+# AdnGlue
+
+AdnGlue is a Maya node that can be used to attach multiple muscles together. This solver uses as input a series of simulated muscles and allows the user to define glue connections between those muscles by using a maximum distance value and painted maps.
+
+The maps required to control the AdnGlue solver can be painted using the standard Maya Paint tool.
+
+### How To Use
+
+The AdnGlue node can be applied to any set of geometries. For each geometry, their mesh at each frame is gathered and simulated to ensure the original distance between the neighboring geometries. This is achieved by creating attachments between each vertex of a geometry and the closest point on the surface of the other geometries. More aspects are then computed to ensure that the simulated output preserves the original shape, edge-lengths and volume of the simulated meshes. The output of the node is a mesh that combines all the input geometries together and adds the results of the solver.
+
+To create an AdnSkin deformer within a Maya scene, the following inputs must be provided:
+
+  - **Input Geometries** (IG): List of geometries to be simulated and attached together by the glue solver.
+
+> [!NOTE]
+> - Applying the AdnGlue node will automatically create a new mesh in your scene, which will display the results of the solver.
```

Comment:
I would add a hint to the naming in case the user can't find it. `_GEO`

Reply:
Added `> - The naming of the new mesh created follows the format "AdnGlue1_GEO".`